lib/l10n/intl_de.arb

```diff
@@ -1053,5 +1053,9 @@
   "sharedWithYou": "Shared with you",
   "sharedByYou": "Shared by you",
   "inviteYourFriendsToEnte": "Invite your friends to ente",
-  "failedToDownloadVideo": "Failed to download video"
+  "failedToDownloadVideo": "Failed to download video",
+  "hiding": "Hiding...",
+  "unhiding": "Unhiding...",
+  "successfullyHid": "Successfully hid",
+  "successfullyUnhid": "Successfully unhid"
```

Comment:
Why are these getting added here? Will it cause incorrect reporting on crowding?